build/test/build-tape.js

```diff
@@ -0,0 +1,10 @@
+var browserify = require('browserify');
+var fs = require('fs');
+
+
+var b = browserify(require.resolve('../../test/util/tape_config.js'), { standalone: 'tape' })
```

Comment:
`rollup` struggles to inject node builtins into tape. 
https://github.com/substack/tape/issues/410
https://github.com/substack/tape/issues/415
And `tape` devs prescribe handling it via browserify on user end.

Modifying `tape` is also proving to be a challenge, mainly because it relies on `through`, the github repo for which is read-only.

Also the use for `browserify` in this case is minimal, since we're using it to only package `tape`, our code still goes through rollup as usual.